deployment/united-manufacturing-hub/values.yaml

```diff
@@ -1512,8 +1512,8 @@ grafana:
     failureThreshold: 10
 
   image:
-    repository: grafana/grafana
-    tag: 8.5.9
+    repository: unitedmanufacturinghub/grafana-umh
+    tag: 1.0.0
```

Comment:
Not really, because this way we can have different versions of `grafana-umh`, whit different version of the plugins in them. Each time we update one of our custom plugins we need to rebuild `grafana-umh` with a new tag to include it with the preinstalled plugins, so having its tag not tied to appVersion is the best idea here. That way if a customer wants to have the latest version of `umh-v2-datasource` he just need to change the tag of `grafana-umh`, without needing to wait for a new version of the whole stack